ci/test/build.py

```diff
@@ -40,5 +48,132 @@ def annotate_buildkite_with_tags(arch: Arch, deps: mzbuild.DependencySet) -> Non
     )
 
 
+def maybe_upload_debuginfo(image: ResolvedImage) -> None:
+    """Uploads debuginfo to `DEBUGINFO_S3_BUCKET` and Polar Signals for all
+    `cargo-bin` binaries that have `split-debuginfo: true`."""
+
+    print(f"Considering whether to upload debuginfo for image {image.name}...")
```

Comment:
I think it's slightly dubious to assume that we will want to upload every binary with split-debuginfo to PS. For example, we don't do this now but I could imagine us wanting to save the debuginfo for `bin/mz` in S3 but not PS.

Reply:
Fair enough. I've adjusted the script to explicitly list the binaries whose debuginfo ought to be uploaded in a new `DEBUGINFO_BINS` constant. Since all binaries today get uploaded to {both S3 and Polar Signals, neither S3 and Polar Signals}, it doesn't seem worth splitting the constant into `DEBUGINFO_S3_BINS` and `DEBUGINFO_POLAR_SIGNALS_BINS`, but the code is now structured in a way that introducing such a split in the future, should it become important, would be straightforward.